detox/android/build.gradle

```diff
@@ -3,12 +3,12 @@ buildscript {
 
     ext {
         isOfficialDetoxLib = true
-        kotlinVersion = '1.8.0'
+        kotlinVersion = '1.9.20'
```

Comment:
That's the new minimum-required version?

Reply:
No, this is what the template wants